x/ethbridge/legacy/v42/migrate.go

```diff
@@ -6,8 +6,8 @@ import (
 )
 
 func Migrate(state v039ethbridge.GenesisState) *types.GenesisState {
-	return  &types.GenesisState{
-		CethReceiveAccount: state.CethReceiverAccount.String(),
-		PeggyTokens:        state.PeggyTokens,
+	return &types.GenesisState{
+		NativeTokenReceiveAccount: state.NativeTokenReceiverAccount.String(),
```

Comment:
This should still be cethReceiver.  v039ethbridge.GenesisState has the data from the 39 version, so it's cethReciver.  We want to convert the old cethRecevier to the new fee receiver.

Reply:
agree, will revert it